### DISCUSSION TITLE
Azure Monitor Resource Graph

### DISCUSSION THREAD

**Discussion by msbannert** [grafana/grafana#64403] [Data Sources]
In the Azure Monitor data source it is now possible to run Azure Graph queries.
However, one needs to specify at least a single subscriptino at which the query gets executed.
It would be cool if one could just configure to run the query against all accessible subscription without the need for specifying the whole list.
In a scenario where the list of available subscriptions might change over time you do not want to edit the dashboards again and again to reflect these changes.
This is by the way also the default behaviour in the azure portal (Azure Resource Graph plugin) : unless you restrict the scope to selected subscriptions the query will return matches for all subscriptions the user has access to.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82793. Feel free to continue the discussion around this there. Thank you!
